### PR TITLE
fix path traversal in mirror-images command

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This fixes https://github.com/kubermatic/kubermatic/security/code-scanning/1

**What type of PR is this?**
/kind bug

**Does this PR introduce a user-facing change? Then add your Release Note here**:
```release-note
Fix potential path traversal in mirror-images command
```

**Documentation**:
```documentation
NONE
```

### CODE REVIEW

File: pkg/install/images/images.go [hunk 1]
```diff
@@ -202,6 +202,11 @@ func extractAddonsFromArchive(dir string, reader *tar.Reader) error {
 
 		path := filepath.Join(dir, header.Name)
 
+		// prevent path traversal (https://cwe.mitre.org/data/definitions/22.html)
+		if strings.Contains(path, "..") {
```
Comment (pkprzekwas) [inline pkg/install/images/images.go:206]: How about stings like `\.\.` ?

Comment (xrstf) [reply pkg/install/images/images.go:206]: Would be news to me that backslashes have any meaning here..

Comment (pkprzekwas) [reply pkg/install/images/images.go:206]: Nevermind, I just checked and go complains about `unknown escape sequence` for an example above.

Comment (xrstf) [reply pkg/install/images/images.go:206]: This is also literally the suggested fix from the security advisory :grin:

**Pull request merged**